air/src/stack/op_flags/mod.rs

```diff
@@ -463,7 +463,7 @@ impl<E: FieldElement> OpFlags<E> {
     /// Operation Flag of READW operation.
     #[inline(always)]
     pub fn readw(&self) -> E {
-        self.degree7_op_flags[get_op_index(Operation::ReadW.op_code())]
+        self.degree7_op_flags[get_op_index(Operation::AdvPopW.op_code())]
```

Comment:
nit: doc-comment for this function should be updated as well.

Reply:
Fixed